src/components/Disclosures/DisclosureControllable.tsx

```diff
@@ -40,17 +42,22 @@ const FacetBrowserDisclosure: FC<FacetBrowserDisclosureProps> = ({
       }`}
       aria-controls={disclosureId}
       aria-expanded={isOpen}
-      onClick={toggleOpen}
-      onKeyDown={toggleOpen}
-      role="button"
-      tabIndex={0}
     >
       <div
         className={clsx(
           "disclosure__headline text-body-large",
           removeHeadlinePadding && "disclosure__headline--no-padding"
         )}
+        onClick={toggleOpen}
+        onKeyDown={toggleOpen}
+        role="button"
+        tabIndex={0}
```

Comment:
can this be opened via keyboard?

Reply:
yup, indeed it can! :)

https://user-images.githubusercontent.com/28546954/206406926-775e0b92-641b-427b-bc87-3a944dd2f42d.mov